src/types.ts

```diff
@@ -30,6 +33,11 @@ export interface IFireOrmQueryLine {
   operator: FirestoreOperators;
 }
 
+export interface IFireOrmOrderBy {
```

Comment:
Since this is kind of internal, I would declare it something like this: 

```typescript
export interface IOrderByParams {
  path: string;
  direction: OrderByDirection;
}
```